### ISSUE TITLE
compare/run UI: Filter run comparison results by z-score or percent change

### ISSUE THREAD

**Issue by alistaire47** [conbench/conbench#1381]
Feedback about the run comparison table:

> it would be useful to filter for significant changes (i.e. `abs(z) >= N` for some hard-coded value of N).

We don't have table filters at the moment, but [datatables does support filtering](https://datatables.net/examples/plug-ins/range_filtering.html) ([or via `filter` method](https://datatables.net/reference/api/filter())) if we want to implement filtering on z-scores, percent change, or other columns. This deserves a conversation about where this functionality would be useful and could supplement searching

**Comment by jgehrcke** [comment]
> it would be useful to filter for significant changes (i.e. abs(z) >= N for some hard-coded value of N).

I agree! But there's a pretty strong alternative for now: sort the column descendingly, and just draw an imaginary line. What's the disadvantage here really (not saying there is none, but I would like for us to put it into words)?

**Comment by alistaire47** [comment]
Agreed, I don't think this is high-priority because there's another way to get there.

One difference: A filter makes it trivial to see how many results you have beyond a particular threshold (or with a particular value[s], if we allow filtering to categorical columns as well). Not an enormous difference, but handy in some contexts.

There's a potentially larger benefit if we allow filtering on all columns: users can keep sorting by z-score (or whatever) while also filtering down to a particular benchmark name (or whatever). Multi-column sorting is another way to get there.
